lib/elasticity/instance_group.rb

```diff
@@ -119,6 +125,7 @@ def to_aws_instance_config
         :instance_count => @count,
         :instance_type => @type,
         :instance_role => @role,
+        :name => @name || "#{@role}-#{object_id}"  # default to role-objectid to differentiate duplicates
```

Comment:
All ruby objects have an object_id. It is the id of particular instance of the ruby class. And thus different for every instance_group in this case